src/pretix/base/forms/questions.py

```diff
@@ -262,6 +262,73 @@ def __init__(self, *args, **kwargs):
                     }
                 )
             )
+        if item.admission and event.settings.attendee_company_asked:
+            self.fields['company'] = forms.CharField(
+                required=event.settings.attendee_company_required and not self.all_optional,
+                label=_('Company'),
+                initial=(cartpos.company if cartpos else orderpos.company),
+            )
```

Comment:
Could we move this up to lines 245 to have the same layout as for our other address-forms?

Company
Name
Address